srcpkgs/gmsl/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gmsl'
+pkgname=gmsl
+version=1.1.8
+revision=1
+noarch=yes
+create_wrksrc=true
+depends="make"
+short_desc="GNU Make Standard Library"
+maintainer="B. Wilson <x@wilsonb.com>"
+license="BSD-3-Clause-Clear"
+homepage="https://sourceforge.net/projects/gmsl/"
+distfiles="${SOURCEFORGE_SITE}/gmsl/${pkgname}-${version}.tar.gz"
+checksum=d569bcf5ec3ce0fd3f9948066ca0bbbf2d25cf529d61dd65e30c941f0eb1adcd
+
+do_extract() {
+	tar -xzf "${XBPS_SRCDISTDIR}/${pkgname}-${version}/${pkgname}-${version}.tar.gz" \
```

Comment:
remove this

Reply:
What is this magic?
The tar doesn't contain a top-level directory and was previously extracting directly into builddir.

Reply:
you set `create_wrksrc` above, it will create the wrksrc and extract into it.

create_wrksrc exists for these situations where the tarball is poorly-made and has no top-level directory.

but since you defined do_extract yourself you made the create_wrksrc not work.